### ISSUE TITLE
How to stop the ## comment block in a PS1.

### ISSUE THREAD

**Issue by k1rb** [PowerShell/vscode-powershell#1286]
While editing a PS1 in VS Code if I type two '#' characters this is happening:
```
    ##############################
    #.SYNOPSIS
    #Short description
    #
    #.DESCRIPTION
    #Long description
    #
    #.PARAMETER Foo
    #Parameter description
    #
    #.PARAMETER Bar
    #Parameter description
    #
    #.EXAMPLE
    #An example
    #
    #.NOTES
    #General notes
    ##############################

```
This is not what I want. How can I stop this from happening?

Thank you for your help.

**Comment by tylerl0706** [comment]
This was fixed in #1282. You will be able to disable it. The release will be out sometime this week. 😀

**Comment by k1rb** [comment]
I appreciate the quick response and thank you guys for all of your hard work!
